rpc/serde/jsonrpc.go

```diff
@@ -146,7 +146,7 @@ type JsonRpcGeneralRequest struct {
 	Jsonrpc string      `json:"jsonrpc"`
 	Id      uint64      `json:"id"`
 	Method  string      `json:"method"`
-	Params  interface{} `json:"params"`
+	Params  interface{} `json:"payload"`
```

Comment:
Does this make sense? Is it a red flag that we are encoding a value under a json key that is different than the struct field?

Reply:
Good catch! I originally renamed `params` to `payload` but then decided to rename it back, and I missed this.

Reply:
Fixed in https://github.com/statechannels/go-nitro/pull/1744/commits/06e6eeb6f7bff7678f4c92262ae39d9b0cdbdc55